composer.json

```diff
@@ -38,7 +38,7 @@
         "sonata-project/easy-extends-bundle": "^2.1",
         "sonata-project/doctrine-extensions": "^1.0",
         "sonata-project/classification-bundle": "^3.0",
-        "knplabs/gaufrette": "^0.1.6",
+        "knplabs/gaufrette": "~0.2",
```

Comment:
Exactly @greg0ire.

`0.x` releases are special, BC break are allowed between minors for this.

This also why I asked @YaoOcelotl to check if this bundle is still works with both versions.